examples/example-app-v18/src/app/demo/demo.spec.ts

```diff
@@ -1,53 +1,67 @@
-import { waitForAsync } from '@angular/core/testing';
+import { Injectable } from '@angular/core';
+import { TestBed, waitForAsync } from '@angular/core/testing';
 import { jest } from '@jest/globals';
 
-import { LightswitchComponent, MasterService, ValueService, ReversePipe } from './demo';
+import { LightswitchComponent, MasterService, ReversePipe, ValueService } from './demo';
 
+@Injectable()
 // eslint-disable-next-line jest/no-export
 export class FakeValueService extends ValueService {
     override value = 'faked service value';
 }
 
 describe('demo (no TestBed):', () => {
```

Comment:
This file can be removed. The purpose of this file was to demonstrate how to run test without involving TestBed. Now we are "forced" to use TestBed so we can remove this